src/DetailsView/Styles/detailsview.scss

```diff
@@ -228,6 +231,74 @@ div.insights-file-issue-details-dialog-container {
         margin-top: 30px;
         font-size: 21px;
     }
+    .footer {
+        margin-bottom: 16px;
+        margin-top: 8px;
+        padding-top: 16px;
+        padding-right: 16px;
+        border-top-style: solid;
```

Comment:
A couple things:
1. You can simplify this to a single property of: border-top.
2. Can we find the value that maps to this color rather than using the color directly?

Reply:
Got it! I couldn't find this color in the Colors.scss--Should I try and add it?

Reply:
Following up with Chelsea--updates to come in next PR!

Reply:
Also collapsing margins and padding to single lines in future PR